x-pack/plugins/cases/common/schema/index.ts

```diff
@@ -22,35 +22,61 @@ export const NonEmptyString = new rt.Type<string, string, unknown>(
   rt.identity
 );
 
+export const limitedStringSchema = (fieldName: string, min: number, max: number) =>
+  new rt.Type<string, string, unknown>(
+    'LimitedString',
+    rt.string.is,
+    (input, context) =>
+      either.chain(rt.string.validate(input, context), (s) => {
+        if (s.trim().length === 0 && s.trim().length < min) {
```

Comment:
As we trim a lot in this function I think it is better if we put it on a variable. For example, `const trimedString = s.trim()`